### PR TITLE
OLMIS-598: Added unit test exemplar and code coverage report

### PR DESCRIPTION
**Pull request by lkukielka**
This change is [Reviewable](https://reviewable.io/reviews/openlmis/openlmis-example/4)

### CODE REVIEW

**Comment by sebbrudzinski** [comment]
@lkukielka Besides the setup, it would be good to have a sample unit test on this repository

**Comment by lkukielka** [comment]
We have _WeatherServiceTest_, do you think we need separate sample test?

**Comment by sebbrudzinski** [comment]
No, if we already have a sample unit test then it's OK

**Comment by joshzamor** [comment]
Any feedback on using record and play in WireMock?

<details><summary>

<i>Previously, sebbrudzinski (Sebastian Brudziński) wrote…</i></summary>



> No, if we already have a sample unit test then it's OK
> </details>

---

Reviewed 4 of 4 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

_[a discussion](https://reviewable.io:443/reviews/openlmis/openlmis-example/4#-KJrZeITKVC_gxJIyQ2U:-KJrZeIUxccL-LnSGZAa:-145924047) (no related file):_
Looks good to merge once typo&merge conflicts resolved.

---

_[README.md, line 50 [r2]](https://reviewable.io:443/reviews/openlmis/openlmis-example/4#-KJrXo_xP6jib5UXJbqi:-KJrXo_ydSqJAIXTcclp:-796971587) ([raw file](https://github.com/openlmis/openlmis-example/blob/b88453eab03d739257a804e7ad736e6c5c12c68e/README.md#L50)):_

> ``` Markdown
> 
> - `clean` to remove build artifacts
> - `build` to build all source. `buil`, after building sources, also runs unit tests. Build will be successful only if all tests pass.
> ```

typo:  build

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openlmis/openlmis-example/4#-:-KJrZTj1TBLrEMvq24ld:1733578788)_

File: build.gradle [hunk 1]
```diff
@@ -54,6 +57,7 @@ flyway {
     placeholderSuffix = ']'
 }
 
+//   Usage: gradle generateMigration [-PmigrationName=name_of_migration]
```
Comment (pgesek) [inline build.gradle]: Wrong place, wrong time

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ This example is meant to show an OpenLMIS 3.x Independent Service at work.
  git clone https://github.com/OpenLMIS/openlmis-example.git
  ```
 2. To properly test this example service is working, enter `spring.mail` values in the `application.properties` file.
-3. Develop w/ Docker by running `docker-compose run --service-ports example`.  
+3. Develop w/ Docker by running `docker-compose run --service-ports example`.
```
Comment (pgesek) [inline README.md:14]: For future refence, do changes like this in separate commits/PRs. It only clutters the actual change

**Pull request merged**